modules/programs/bash/default.nix

```diff
@@ -21,14 +25,18 @@ in
       type = types.lines;
     };
 
-    programs.bash.enableCompletion = mkOption {
-      type = types.bool;
-      default = false;
-      description = ''
-        Enable bash completion for all interactive bash shells.
+    programs.bash.completion = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Enable bash completion for all interactive bash shells.
+
+          NOTE. This doesn't work with bash 3.2, which is the default on macOS.
```

Comment:
Is this still the default on macOS?

Reply:
macOS will never ship a newer Bash since Apple refuses to touch GPLv3, though it’s not the default *login shell* any more.

Reply:
No bash is not the default and has not been for 10? years. The default now is zsh

Reply:
```suggestion
          NOTE: This doesn't work with bash 3.2, which is installed by default on macOS by Apple.
```